app/src/components/v-form/v-form.vue

```diff
@@ -296,8 +296,8 @@ function useForm() {
 
 	function setPrimaryKeyReadonly(field: Field) {
 		if (
-			field.schema?.is_generated === true &&
-			(field.schema?.has_auto_increment === true || (field.schema?.is_primary_key === true && props.primaryKey !== '+'))
+			field.schema?.is_primary_key === true &&
+			(field.schema?.has_auto_increment === true || props.primaryKey !== '+')
```

Comment:
Although this does "revert" #14947, but I believe we will still need to fix the original "not able to edit manual primary key" issue that #14947 aimed to fix 👍 

Example test case:

https://user-images.githubusercontent.com/42867097/210757331-99887f9e-8d74-45d8-9bea-d8707db188b9.mp4

Reply:
You're very right as usual 😄 I have updated the readonly check to only target uuid and auto increment IDs in line with the changes done for deleting the id

Reply:
Making the check inline with the changes in use-items makes perfect sense! 👍